app.js

```diff
@@ -1,10 +1,18 @@
 // Load up the discord.js library
 const Discord = require("discord.js");
+
 const client = new Discord.Client();
+
 const config = require("./config.json");
 const gotkicked = require("./gotkicked.json");
 const joinmessages = require("./joinmessages.json");
 const help = require('./help.json');
+
+var colors = new Array();
+colors.red = 0x781706;
```

Comment:
but then using string indexes converts the array to an object right afterward
id recommend writing it like 
```
var colors = {
    red:  0x781706,
    orange: 0xBA430D,
    green: 0x037800
};
```